core/src/block.rs

```diff
@@ -340,39 +340,39 @@ impl ChainedBlock {
 declare_versioned_with_scale!(VersionedValidBlock 1..2, Debug, Clone, iroha_macro::FromVariant, IntoSchema);
 
 impl VersionedValidBlock {
-    /// Same as [`as_v1`](`VersionedValidBlock::as_v1()`) but also does conversion
+    /// Converts from `&VersionedValidBlock` to V1 reference
```

Comment:
```suggestion
    /// Converts from [`&VersionedValidBlock`] to V1 reference
```

Reply:
I actually looked into [`Option`](https://doc.rust-lang.org/std/option/enum.Option.html#method.as_ref) documentation before making this change. I guess they didn't make it a link because the return type of the method is already a link